offClick.js

```diff
@@ -30,7 +35,19 @@ angular.module('offClick', [])
         return false;
     }
 
+    function touchStartEventHandler(){
+      touchMove = false;
```

Comment:
Some devices have both a touch screen and a traditional pointer (laptop with a touchscreen).  In that case, the user match touch, and then continue with clicking.  When that happens, this will not get called and `touchMove` will never be set back to `false`.
It would probably be best to set `touchMove` to false below where you are checking if it is `true` and returning `false`.